ui/app/components/app/signature-request/signature-request-header/signature-request-header.component.js

```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import AccountDropdownMini from '../../../ui/account-dropdown-mini'
+import NetworkDisplay from '../../network-display'
+
+export default class SignatureRequestHeader extends PureComponent {
+  static propTypes = {
+    selectedAccount: PropTypes.object.isRequired,
+    accounts: PropTypes.object,
+  }
+
+  render () {
+    const { selectedAccount, accounts } = this.props
+    return (
+      <div className="signature-request-header">
+        <div className="signature-request-header--account">
+          {selectedAccount && accounts && <AccountDropdownMini
+            selectedAccount={selectedAccount}
```

Comment:
Note that this `selectedAccount` is the current account. I think you mentioned you weren't sure where to get that from state